### PR TITLE
Add configurable delay for initialization

### PR DESCRIPTION
**Pull request by juhasch**
Workaround for problems like #988, when the gutter is not displayed correctly.
Add a configuration option for a delay in ms.

### CODE REVIEW

File: src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding.yaml [hunk 1]
```diff
@@ -10,3 +10,8 @@ Parameters:
   description: Hotkey to fold/unfold code
   input_type: hotkey
   default: Alt-F
+- name: init_delay
```
Comment (jcb91) [inline src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding.yaml]: the name used in the js is `init_timeout`!

File: src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml [hunk 2]
```diff
@@ -9,4 +9,10 @@ Link: readme.md
 Icon: codefolding_editor.png
 Main: edit.js
 Compatibility: 4.x, 5.x
+Parameters:
+- name: init_delay
```
Comment (jcb91) [inline src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml]: again, the name used in the js is `init_timeout`

**Comment by jcb91** [comment]
Good plan, just a couple of typos, I think

File: src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml (hunk 2 above)
Comment (juhasch) [inline src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml]: Thanks for catching. I somehow reverted the change I did to the JS file.

File: src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding.yaml (hunk 1 above)
Comment (juhasch) [inline src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding.yaml]: Yep.

**Comment by jcb91** [comment]
looks good to me :+1:

**Pull request merged**